common/changes/@uifabric/example-app-base/tooltipHost-onTooltipToggle_2017-08-11-16-12.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/example-app-base",
+      "type": "none"
```

Comment:
(it doens't hurt anything to include them but it can reduce the changes in your pr.)